applications/updater/util/update_task_workers.c

```diff
@@ -0,0 +1,144 @@
+#include "update_task.h"
+#include "update_task_i.h"
+
+#include <furi.h>
+#include <furi_hal.h>
+#include <storage/storage.h>
+#include <toolbox/path.h>
+#include <update_util/dfu_file.h>
+#include <update_util/lfs_backup.h>
+#include <update_util/update_hl.h>
+
+#define CHECK_RESULT(x) \
+    if(!(x)) {          \
+        break;          \
+    }
+
+static const DfuValidationParams flipper_dfu_params = {
+    .device = 0xFFFF,
+    .product = 0xDF11,
+    .vendor = 0x0483,
```

Comment:
These are magic numbers, wizard.

Reply:
Also, if you have a "descriptor" struct, that just holds some immutable values and declared globally, each field must be constant.

Reply:
It is named `flipper_dfu_pararms` for a reason.

Reply:
```
#define ST_VENDOR_ID 0x0483
#define ST_F4_PRODUCT_ID 0xDF11
```
It will be much clearer where these constants come from.

Reply:
Fixed